docs/it/federation_endpoint.rst

```diff
@@ -70,24 +70,20 @@ Fetch Entity Statements Request
 La richiesta DEVE essere una richiesta HTTP che usa il metodo GET ed ha i seguenti parametri di query:
 
 .. list-table:: 
-   :widths: 20 20 40 20
+   :widths: 20 60 20
    :header-rows: 1
 
    * - **Claim**
-     - **Tipo**
      - **Descrizione**
      - **Obbligatorio**
    * - **iss**
-     - String
-     - L’identificatore di entità dell’emettitore dal quale si vuole l’emissione dell’ES. A causa della normalizzazione dell’URL, più emettitori POSSONO puntare ad un Fetch Endpoint condiviso. Questo parametro esplicita esattamente da quale emettitore si vuole l’ES. Senza questo parametro l’emettitore coincide con l’entità alla quale è stata fatta la richiesta.
+     - String. L’identificatore di entità dell’emettitore dal quale si vuole l’emissione dell’ES. A causa della normalizzazione dell’URL, più emettitori POSSONO puntare ad un Fetch Endpoint condiviso. Questo parametro esplicita esattamente da quale emettitore si vuole l’ES. Senza questo parametro l’emettitore coincide con l’entità alla quale è stata fatta la richiesta.
```

Comment:
usare il simbolo '

Reply:
Se uso il simbolo ', l'html che ne risulta lo sostituisce col simbolo ", che non c'entra niente con l'apostrofo. È per questo che avevo scelto ’. Ma se vuoi, lo faccio lo stesso.

Reply:
impostato
smartquotes = False
in conf.py
ora possiamo usare i caratteri ' e "